python/Ganga/GPIDev/Lib/GangaList/GangaList.py

```diff
@@ -52,8 +52,8 @@ def makeGangaListByRef(_list, preparable=False):
     result = GangaList()
     if len(_list) == 0:
         return result
-    temp_list = [stripProxy(element) for element in _list]
-    result._list = temp_list
+
+    result._list.extend([stripProxy(element) for element in _list])
```

Comment:
Hmm, well historically it looked like:
```python
def makeGangaListByRef(_list):
    """Faster version of makeGangaList. Does not make a copy of _list but use it by reference."""
    result = GangaList()
    result._list = _list
    return result
```
which I feel has the correct sematics. Now, this doesn't work with the current codebase due to some funky interception of the assignment to `_list`. If possible I would like to get back to this working in this way.